client/public/index.hbs

```diff
@@ -37,6 +37,20 @@
 			dictionary : "${serializedDictionary?no_esc}"
 		}
 		</script>
-
+    <script>
+      const isIE = /*@cc_on!@*/false || !!document.documentMode;
```

Comment:
minor point - can we move this to `polyfills.js`?

Reply:
after moving this to `polyfills.js` two things came up. First, LGTM is producing a warning about conditional comments. Second, i had to add `--testEnvironment=jsdom` argument to: 

```
"test-jest": "export ANET_CONFIG=../anet.yml ; jest --testPathPattern='tests/(actions|reducers)' --testEnvironment=jsdom"
```

because jest tests were unable to run.

Reply:
Re: `jsdom`: in the `keycloak-integration` branch I had the same thing, and made this commit:
```
commit 8a51c98f85b434ed20e777d6decde4dd62314f4e
Author: Gertjan van Oosten <gertjan@west.nl>
Date:   Mon May 25 17:30:25 2020 +0200

    NCI-Agency/anet#2314: Update for Jest tests

diff --git a/client/package.json b/client/package.json
index f256c263d..7b67d5d23 100644
--- a/client/package.json
+++ b/client/package.json
@@ -193,7 +193,7 @@
     "testPathIgnorePatterns": [
       "<rootDir>/(build|docs|node_modules)/"
     ],
-    "testEnvironment": "node"
+    "testEnvironment": "jsdom"
   },
   "browserslist": [
     "> 1%",
```
I suggest you make the same change in this branch. It will get to `candidate` eventually. :wink:

Reply:
Thanks for the heads up :+1:

Reply:
About the LGTM warning;
`/*@cc_on!@*/false || !!document.documentMode` detects IE6-11. By removing the conditional comment and using only `document.documentMode` we can detect IE 8-11. Would this be sufficient?
More information about `documentMode` can be found [here](https://docs.microsoft.com/en-us/internet-explorer/ie11-deploy-guide/deprecated-document-modes).

Reply:
As much as I like clever hacks :smile: why not use something like https://www.npmjs.com/package/check-ie ?

Reply:
IE detection is a nightmare :D I used [this library](https://github.com/DamonOehlman/detect-browser) at first. Then I decided not to use an extra dependency for such a simple feature. I moved to a pure js and simple solution.

While doing research about browser detection, everywhere it is stated that user agent strings aren't reliable for browser detection since they can be spoofed easily. It is advised to use feature detection such as `document.documentMode`.

`/*@cc_on!@*/false || !!document.documentMode` seemed to be the easiest and reliable solution until LGTM started complaining about it :)

Reply:
Understood. But what we're trying to do here is a bit simpler than browser _feature_ detection; it is to discourage users from visiting ANET with Internet Explorer. Anyone pretending to be IE when they're really not (or the other way around, probably a lot more difficult) will get what's coming to them. :laughing: 
Simply using `change-ie` would mean it can also be tested with e.g. Google Chrome:
![Screenshot from 2020-06-11 15-29-06](https://user-images.githubusercontent.com/445238/84390855-70be9a00-abf8-11ea-846e-ea85dfa3f4ce.png)